docs/integrations/sources/thinkific.md

```diff
@@ -0,0 +1,35 @@
+# Thinkific
+Airbyte connector for Thinkific, allowing you to seamlessly sync data like users, course participants, and instructors from Thinkific to other platforms. It&#39;s designed to make managing and analyzing your online courses and communities even easier!
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+| `subdomain` | `string` | subdomain. superb |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| courses | id | DefaultPaginator | ✅ |  ❌  |
```

Comment:
Can you include `courses_reviews` endpoint as well?

Reply:
there is a stream named reviews which uses `courses_reviews ` end point, should I rename it